### PR TITLE
doc: fixed broken link in Swift Settings section

### PR DESCRIPTION
**Pull request by falcon78921**
- [x] References tracker ticket
- [x] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

Fixes: https://tracker.ceph.com/issues/40575
Signed-off-by: James McClune <jmcclune@mcclunetechnologies.net>

### CODE REVIEW

**Comment by falcon78921** [comment]
jenkins render docs

**Comment by joscollin** [comment]
Now you have introduced a url for `(`

File: doc/radosgw/config-ref.rst [hunk 1]
```diff
@@ -612,9 +612,9 @@ Swift Settings
 
 	      The ``X-History-Location`` attribute, also understood by
 	      OpenStack Swift for handling ``DELETE`` operations
-	      `slightly differently
-	      <https://docs.openstack.org/swift/latest/overview_object_versioning.html>`_
-	      from ``X-Versions-Location``, is currently not
+	      `(slightly different
```
Comment (tchaikov) [inline doc/radosgw/config-ref.rst]: probably, instead, we can link "OpenStack Swift" to https://docs.openstack.org/swift/latest/api/object_versioning.html. like
```rst
A slightly different attribute, ``X-History-Location``, which is also understood by 
`OpenStack Swift <https://docs.openstack.org/swift/latest/api/object_versioning.html>`
for handling ``DELETE`` operations, is currently not supported.
```

Comment (falcon78921) [reply doc/radosgw/config-ref.rst]: @tchaikov yeah, that sounds much better :)

Comment (falcon78921) [reply doc/radosgw/config-ref.rst]: @tchaikov I made the change.

**Comment by falcon78921** [comment]
jenkins render docs

File: doc/radosgw/config-ref.rst [hunk 2]
```diff
@@ -610,12 +610,9 @@ Swift Settings
...
               mechanism.
 
-	      The ``X-History-Location`` attribute, also understood by
-	      OpenStack Swift for handling ``DELETE`` operations
-	      `slightly differently
-	      <https://docs.openstack.org/swift/latest/overview_object_versioning.html>`_
-	      from ``X-Versions-Location``, is currently not
-	      supported.
+	      A slightly different attribute, ``X-History-Location``, which is also understood by 
+              `OpenStack Swift <https://docs.openstack.org/swift/latest/api/object_versioning.html>`
```
Comment (tchaikov) [inline doc/radosgw/config-ref.rst]: @falcon78921 you need to add an underscore after the backquote .

Comment (tchaikov) [reply doc/radosgw/config-ref.rst]: see http://docs.ceph.com/ceph-prs/28774/radosgw/config-ref/

Comment (falcon78921) [reply doc/radosgw/config-ref.rst]: @tchaikov Oops, missed that. I added the backquote and tested on my end. Should be good to go :)

**Comment by falcon78921** [comment]
jenkins render docs

**Pull request merged**